api-report/tree.api.md

```diff
@@ -351,6 +369,11 @@ export interface EditableTreeContext extends ISubscribable<ForestEvents> {
     set unwrappedRoot(data: ContextuallyTypedNodeData | undefined);
 }
 
+// @alpha
+export interface EditableTreeEvents {
+    changed(): void;
```

Comment:
There's an inconsistency in tense between 'changed' vs. 'fooChange'.

.NET used "-ing" for before events and "-ed" for after events.  (changing = about to change, changed = has changed)

Reply:
this one is neither before or after since both the anchorset and forest are explicitly not in a consistent state when its called.

I think using the present progressive better implies this, so I have renamed them and improved the clarity in the docs a bit.